.travis/yapf.sh

```diff
@@ -9,17 +9,49 @@ builtin cd "$(dirname "${BASH_SOURCE:-$0}")"
 ROOT="$(git rev-parse --show-toplevel)"
 builtin cd "$ROOT"
 
-yapf \
-    --style "$ROOT/.style.yapf" \
-    --in-place --recursive --parallel \
-    --exclude 'python/ray/cloudpickle' \
-    --exclude 'python/ray/dataframe' \
-    --exclude 'python/ray/rllib' \
-    -- \
-    'test' 'python'
+YAPF_FLAGS="--style $ROOT/.style.yapf --in-place --recursive --parallel"
+
+YAPF_EXCLUDES="--exclude 'python/ray/cloudpickle' \
+--exclude 'python/ray/dataframe' \
+--exclude 'python/ray/rllib' \
+--exclude 'python/build' \
+--exclude 'python/ray/pyarrow_files' \
+--exclude 'python/ray/core/src/ray/gcs' \
+--exclude 'python/ray/common/thirdparty'"
+
+UPSTREAM_MASTER=${RAY_UPSTREAM_BRANCH:-upstream/master}
```

Comment:
This doesn't need an env variable, and `upstream/master` isn't always defined (my remotes are `alok` and `origin`). Setting this to `master` or `origin/master` is a better idea because `master` is meant to track the main branch.

Reply:
Changed to `origin/master`. Note that you can override the target here by setting `RAY_UPSTREAM_BRANCH`.

Reply:
Actually there's a better way to do this... WIP

Reply:
Ok... now we always fetch the latest upstream, and grab the merge base, so the diff should always be correct.